File: data/fireworks/docs/code_example.py
import openai

client = openai.OpenAI(
    base_url="{{ endpoint }}",
    api_key="{{ api_key }}",
)
response = client.chat.completions.create(
    model="{{ model_name }}",
    # Change the model name to the deepseek one
    messages=[
        {
            "role": "user",
            "content": "Say this is a test",
        }
    ],
)
print(response.choices[0].message.content)

File: data/fireworks/docs/code_example.js
import OpenAI from "openai";

const openai = new OpenAI({
  baseURL: "{{ endpoint }}",
  apiKey: "{{ api_key }}",
});

async function main() {
  const completion = await openai.chat.completions.create({
    model: "{{ model_name }}",
    messages: [{ role: "user", content: "Say this is a test" }],
  });

  console.log(completion.choices[0].message.content);
}

main();
